worker/manual-jobs.ts

```diff
@@ -1,19 +1,30 @@
+import { isFantomNetwork } from '../modules/config/network-config';
+import { createAlertsIfNotExist } from './create-alerts';
+import { fantomJobs } from './fantom-jobs';
+import { optimismJobs } from './optimism-jobs';
 import { workerQueue } from './queue';
 
-export type WokerJobType = 'sync-pools' | 'user-sync-wallet-balances-for-all-pools' | 'user-sync-staked-balances';
-
 export type WorkerJob = {
-    type: WokerJobType;
+    name: string;
+    interval: number;
 };
 
-async function scheduleJobWithInterval(job: WokerJobType, intervalMs: number): Promise<void> {
-    await workerQueue.sendWithInterval(JSON.stringify({ type: job }), intervalMs);
+async function scheduleJobWithInterval(job: string, intervalMs: number): Promise<void> {
+    await workerQueue.sendWithInterval(JSON.stringify({ name: job }), intervalMs);
 }
 
-// all jobs requiring manual scheduling will be handled here (e.g. sub minute crons)
+export async function scheduleJobs(): Promise<void> {
+    if (isFantomNetwork()) {
+        fantomJobs.forEach(async (job) => await scheduleJobWithInterval(job.name, job.interval));
```

Comment:
this works, but might not exactly do what you expect, read this https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop